### DISCUSSION TITLE
Welcome to Screenbox Discussions!

### DISCUSSION THREAD

**Discussion by huynhsontung** [huynhsontung/Screenbox#295] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by akasharumugamm** [comment]
Please provide option to reduce subtitle size

**Reply by huynhsontung** [reply]
Hello, 
Subtitle styling customization is being tracked in #143. For now, you can change the subtitle font size by using the "Advanced mode" in the settings. Add `--freetype-rel-fontsize=10` into "Global arguments", where 10 is the font size value that you can customize. The bigger the value the smaller the text.
